infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -559,7 +535,7 @@ private LogData getLogData(LogEntry entry) {
     private Metadata parseMetadata(FileChannel fileChannel, String segmentFile) throws IOException {
         long actualMetaDataSize = fileChannel.size() - fileChannel.position();
         if (actualMetaDataSize < METADATA_SIZE) {
-            log.error("Metadata has wrong size. Actual size: {}, expected: {}",
+            log.warn("Metadata has wrong size. Actual size: {}, expected: {}",
```

Comment:
Just a question, why is this not considered an ERROR?

Reply:
tbh I am not sure why this particular line shows up in the log but it does show up all the time (I did not have time to investigate). I was told that sometimes other verticals attribute errors to us due to this line but nothing has happened because of it so far. I am reducing the severity to warn here but still leaving it for future reference to investigate further.